src/DynamoPackages/PackageManagerClient.cs

```diff
@@ -243,7 +260,7 @@ internal PackageManagerResult Undeprecate(string name)
         internal bool DoesCurrentUserOwnPackage(Package package,string username) 
         {
             var pkg = new PackageInfo(package.Name, new Version(package.VersionName));
-            var header = GetPackageHeader(pkg);
+            var header = GetPackageMaintainers(pkg);
```

Comment:
same question about this variable name.

Reply:
I guess renaming this variable to `maintainers` will make sense

Reply:
`maintainers` and `maintainer` are already used in the inline query below this, how about `mnt`?

Reply:
sounds good.